apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1664,6 +1664,71 @@ describe('Ticket', () => {
     })
 
     describe('Validations', () => {
+        describe('guards', () => {
+            test('user: resident should not be able to create tickets with identical text over the limit', async () => {
+                const details = 'I have some problems with hot water!'
+
+                // User, should be banned from creating tickets to organization if he exceeds the limits.
+                const userClient = await makeClientWithResidentAccessAndProperty()
+                const unitName = faker.random.alphaNumeric(5)
+                await createTestResident(admin, userClient.user, userClient.property, {
+                    unitName,
+                })
+                for (let i = 0; i < DAILY_SAME_TICKET_LIMIT; ++i) {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName, details })
+                }
+
+                await expectToThrowGQLError(async () => {
+                    await createTestTicket(userClient, userClient.organization, userClient.property, { unitName, details })
+                }, ERRORS.SAME_TICKET_FOR_PHONE_DAY_LIMIT_REACHED)
+
+                // User, banned from creating tickets to organization 1, can still create tickets to organization 2
+                const client2 = await makeClientWithProperty()
```

Comment:
Can you rename it please to make a bit more clear? `userClient` and `client2` IMO should mean similar thing, but one of them is resident, and second one is Organization admin, which is kinda opposite.